index.html

```diff
@@ -5,14 +5,81 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Quiz App</title>
     <link rel="stylesheet" href="styles/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link href="/fonts/css/all.css" rel="stylesheet" />
   </head>
   <body class="app-grid">
-    <header class="app-grid__header">
-
-      <h1 class="">My Quiz App</h1>
-      
+    <header class="header">
+      <h1 class="header__heading">My Quiz App</h1>
     </header>
-    <main class="app-grid__main"></main>
-    <footer class="app-grid__footer"></footer>
+    <main class="main">
+      <section class="main__section">
+        <h2 class="main__heading">Question</h2>
+        <p class="main__text">Wie viele Menschen leben in Deutschland?</p>
+        <details class="main__details">
+          <summary class="main__summary">Show answer</summary>
+          <p class="main__text">
+            Es leben ca 83 Millionen Menschen in Deutschland.
+          </p>
+        </details>
+        <ul class="main__list">
+          <li class="main__li--item">politics</li>
+          <li class="main__li--item">geography</li>
+          <li class="main__li--item">germany</li>
+        </ul>
+        <div class="main__bookmark-on"><i class="fas fa-bookmark"></i></div>
+      </section>
+      <section class="main__section">
+        <h2 class="main__heading">Question</h2>
+        <p class="main__text">Wie viele Menschen leben in Deutschland?</p>
+        <details class="main__details">
+          <summary class="main__summary">Show answer</summary>
+          <p class="main__text">
+            Es leben ca 83 Millionen Menschen in Deutschland.
+          </p>
+        </details>
+        <ul class="main__list">
+          <li class="main__li--item">politics</li>
+          <li class="main__li--item">geography</li>
+          <li class="main__li--item">germany</li>
+        </ul>
+        <div class="main__bookmark-off"><i class="fas fa-bookmark"></i></div>
+      </section>
+    </main>
+    <div class="fixed">
```

Comment:
In your `app-grid` you defined a grid with `grid-template-columns: 48px auto 48px;`. Because you set this, the last element in your body, for example a `footer`-element is _automatically_ fixed to the bottom, if you give the `main`-element `overflow: scroll;`. This way you don't need positioning to fix your footer.
```suggestion
    <footer>
```